### DISCUSSION TITLE
Disabling device previews

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#9754] [Ideas]
The device previews added in 3.6 are great, but it'd be nice if it was possible to turn them off (including the "Rotate" button) on a per-installation basis, perhaps via a config setting like `enableDevicePreviews` or similar (default `true`, of course). Alternatively, and possible more ideal; making it possible to disable them on a per-preview target basis, similar to the "Auto-refresh" setting.

Craft CMS can be used to power pretty much anything, so being able to toggle between desktop, tablet and mobile viewports in Preview isn't _always_ relevant. For example, we've created a few kiosk apps using headless Craft, where the device previews are completely redundant and potentially confusing to the content authors. This isn't the biggest thing in the world, obviously, but I can imagine there are other use cases where hiding/disabling the device previews would be preferable.

**Comment by brandonkelly** [comment]
I wonder if it would be better if the list was just configurable? Maybe in your case, it would be best if the viewport was limited to a single specific viewport size?

**Reply by mmikkel** [reply]
That’s probably a better idea!
